### ISSUE TITLE
Wax error for Guid value

### ISSUE THREAD

**Issue by TimBakkerTooling** [tom-englert/Wax#67]
When creating the Wix project file with Wax Setup editor i get the following compilation error:

The Component/@Guid attribute's value, '1c3291b8-f364-4d4b-beb0-26f8fa743e78', is a mixed-case guid.  All letters in a guid value should be uppercase.

**Comment by tom-englert** [comment]
I never saw such an Error. Maybe you can turn off this error?

**Comment by TimBakkerVBH** [comment]
Yes i found out that the error message level was "Pedantic". Setting back to normal fixed the issue.
However, i fixed the error by adding .ToUpper() at the file creating method.

**Comment by tom-englert** [comment]
Can you provide a PR?

**Comment by TimBakkerVBH** [comment]
I just created a Pull Request! I'm not sure what the correct versionnumbering is.

Also fixed another error about KeyPaths

**Comment by tom-englert** [comment]
The KeyPath is not really required here, but having it explicit is also fine.

**Comment by Miles-Davies-HORIBA** [comment]
@tom-englert the reason for the case needing to be upper case is detailed [here](https://docs.microsoft.com/en-gb/windows/win32/msi/component-table?redirectedfrom=MSDN)

> Note that the letters of these GUIDs must be uppercase. Utilities such as GUIDGEN can generate GUIDs containing lowercase letters. The lowercase letters must be changed to uppercase to make these valid component code GUIDs.

While it is OK to not have the KeyPath flag set it is more correct to do so. Especially when there is one file per component.
> If the value is not null, then KeyPath is either a primary key into the Registry, ODBCDataSource, or File tables depending upon the Attribute value. If KeyPath is null, then the folder of the Directory_ column is used as the key path.

For file components this should probably be set unless the file is protected by WRP (not commonly the case where Wax is used).
